tests/test_url.py

```diff
@@ -95,14 +100,15 @@ def url_quote(url):
 
     def test_norm_quote(self):
         # Test url norm quoting.
-        url = "http://groups.google.com/groups?hl=en&lr&ie=UTF-8&"\
-              "threadm=3845B54D.E546F9BD%40monmouth.com&rnum=2&"\
-              "prev=/groups%3Fq%3Dlogitech%2Bwingman%2Bextreme%2Bdigital"\
-              "%2B3d%26hl%3Den%26lr%3D%26ie%3DUTF-8%26selm%3D3845B54D.E5"\
-              "46F9BD%2540monmouth.com%26rnum%3D2"
+        url = (
+            "http://groups.google.com/groups?hl=en&lr&ie=UTF-8&"
+            "threadm=3845B54D.E546F9BD%40monmouth.com&rnum=2&"
+            "prev=/groups%3Fq%3Dlogitech%2Bwingman%2Bextreme%2Bdigital"
+            "%2B3d%26hl%3Den%26lr%3D%26ie%3DUTF-8%26selm%3D3845B54D.E5"
+            "46F9BD%2540monmouth.com%26rnum%3D2"
+        )
         self.urlnormtest(url, url)
-        url = "http://redirect.alexa.com/redirect?"\
-              "http://www.offeroptimizer.com"
+        url = "http://redirect.alexa.com/redirect?" "http://www.offeroptimizer.com"
```

Comment:
```suggestion
        url = "http://redirect.alexa.com/redirect?http://www.offeroptimizer.com"
```

Reply:
`git grep '" "'` tells me I missed another one like this, in this file:
```python
url = "http://server/..%5c..%5c..%5c..%5c..%5c..%5c..%5c.." "%5ccskin.zip"
```

Reply:
Done.